app/middleware/account_elevator.rb

```diff
@@ -17,7 +17,7 @@ def self.switch!(cname)
     elsif Account.any?
       raise "No tenant found for #{cname}"
     else
-      logger.info "It looks like we're in single tenant mode. No tenant found for #{cname}"
+      Rails.logger.info "It looks like we're in single tenant mode. No tenant found for #{cname}"
```

Comment:
this was unrelated, but is a very long standing bug - logger is never defined in this file.